src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -65,7 +93,8 @@ trait EntityComponent {
 
     /** gets the given entity */
     def get(workspaceContext: SlickWorkspaceContext, entityType: String, entityName: String): ReadAction[Option[Entity]] = {
-      unmarshalEntities(joinOnAttributesAndRefs(findEntityByName(workspaceContext.workspaceId, entityType, entityName))).map(_.headOption)
+      val sql = sql"""#$baseEntityAndAttributeSql where e.#${quoteIdentifier("name")} = ${entityName} and e.#${quoteIdentifier("entity_type")} = ${entityType} and e.#${quoteIdentifier("workspace_id")} = ${workspaceContext.workspaceId.toString}""".as[EntityListResult]
```

Comment:
I don't know how to do this because of the string interpolation. How does one do string interpolation on a non-literal string?